### ISSUE TITLE
Search icon (magnifying glass) is not rendered properly in FF

### ISSUE THREAD

**Issue by sophokles73** [google/docsy#1256]
The search input field's magnifying glass is not rendered as the magnifying glass icon but as the literal text `fl` in my browser. See attached screenshot

![image](https://user-images.githubusercontent.com/5682135/194233797-30ef321c-f388-49b6-97ac-b9995cd6e767.png)

This seems to be due to the order in which the font-family is defined for the `.td-search-input` class in `_search.scss`
```css
.td-search-input {
    ...
    font-family: $font-family-base, $font-awesome-font-name;
}
```

where the font awesome font is added to the end of the list but one of the fonts before font awesome in the list already contains a value for the `&#xf002;` entity.

---

**Edit** (@chalin):

- [x] Fix for regular search - #1247 
- [ ] Fix for offline search

**Comment by sophokles73** [comment]
BTW, this occurs when I open the Docsy user guide at https://www.docsy.dev/docs/

My browser is Firefox 105.0.2 on Ubuntu 22.04

**Comment by LisaFC** [comment]
Huh, that's weird. It's showing up as the magnifying glass icon for me (Chrome on Mac). Any other users with other browsers/OS seeing it?  @chalin any ideas?

**Comment by sophokles73** [comment]
It also displays properly on my Android 11 based smart phone running Firefox ...

**Comment by chalin** [comment]
Thanks @sophokles73.

Which version of Docsy are you using?

**Comment by chalin** [comment]
Ah, I'm seeing that problem in FF 105.0.2 under macOS. No issue under Chrome or Safari.

**Comment by chalin** [comment]
Helvetica Neue is the culprit under FF. The search box will need to be reworked to fix this. I'll give it a try tomorrow or this w/e. @geriom - do you want to wait for this fix?

**Comment by sophokles73** [comment]
@chalin I am using the HEAD revision of the main branch.

**Comment by chalin** [comment]
Hey, all: there was a solution that pre-dated even this issue! :) 

- #1247

**Comment by chalin** [comment]
I've confirmed that the fix (#1247) works in FF as well as Chrome and Safari.

**Comment by chalin** [comment]
Reopening since this needs to be fixed for offline search too!
